node/src/types/json_compatibility/stored_value.rs

```diff
@@ -44,7 +44,9 @@ pub enum StoredValue {
     /// A bid
     Bid(Box<Bid>),
     /// A withdraw
-    Withdraw(Vec<UnbondingPurse>),
+    Withdraw(Vec<WithdrawPurse>),
```

Comment:
Won't this change the already-existing events/data schemes? I would expect that JSON representation is derived via `serde-json` + `Serialize` and that would autogenerate the names from the types/variants?

/cc @Fraser999